src/administration/from_scratch.md

```diff
@@ -26,52 +87,74 @@ sudo mv magick /usr/bin/
+
+### Lemmy Backend
+
+#### Build the backend
+
+Create user account on Linux for the lemmy_server application
+
+```
 sudo adduser lemmy --system --disabled-login --no-create-home --group
 ```
 
-Tune your PostgreSQL settings to match your hardware via https://pgtune.leopard.in.ua/#/
-
 Compile and install Lemmy, given the from-scratch intention, this will be done via GitHub checkout. This can be done by a normal unprivledged user (using the same Linux account you used for rustup).
 
 ```bash
-# protobuf-compiler may be required for Ubuntu 22.04.2 installs, please report testing in lemmy-docs issues
-sudo apt install protobuf-compiler
 git clone https://github.com/LemmyNet/lemmy.git lemmy
 cd lemmy
-git checkout 0.18.2
+git checkout 0.18.5
 git submodule init
 git submodule update --recursive --remote
 echo "pub const VERSION: &str = \"$(git describe --tag)\";" > "crates/utils/src/version.rs"
-# These instructions assume you build pictrs independent, but it is
-# OPTIONAL on next command: --features embed-pictrs
+```
+
+When using the embedded pict-rs, use the following build command:
+
+```
+cargo build --release --features embed-pictrs
+```
+
+Otherwise, just move forward with the following.
+
+```
 cargo build --release
-# copy compiled binary to destination
-sudo cp target/release/lemmy_server /usr/bin/lemmy_server
 ```
 
-Note:
+#### Deployment
+
+Because we should [follow the Linux way](https://tldp.org/LDP/Linux-Filesystem-Hierarchy/html/opt.html), we should use the `/opt` directory to colocate the backend, frontend and pict-rs.
+
+```
+sudo mkdir /opt/lemmy
+sudo mkdir /opt/lemmy/lemmy-server
+sudo mkdir /opt/lemmy/pictrs
+sudo mkdir /opt/lemmy/pictrs/files
+sudo mkdir /opt/lemmy/pictrs/sled-repo
+sudo mkdir /opt/lemmy/pictrs/old
+sudo chown -R lemmy:lemmy /opt/lemmy
+```
+
+Note that it might not be the most obvious thing, but **creating the pictrs directories is not optional**.
+
+Then copy the binary.
+
+```
+sudo cp target/release/lemmy_server /opt/lemmy-server/lemmy_server
+```
 
-- Lemmy currently only supports non-SSL connections to databases. More info [here](https://github.com/LemmyNet/lemmy/issues/3007).
-- Your postgres config might need to be edited to allow password authentication instead of peer authentication. Simply add:
-  ```
-  local   lemmy           lemmy                                   md5
-  # Add the line above.
-  # Do not add the line below, it should already exist in your pg_hba.conf in some form.
-  local   all             all                                     peer
-  ```
+#### Configuration
 
-Minimal Lemmy config, put this in `/etc/lemmy/lemmy.hjson` (see [here](https://github.com/LemmyNet/lemmy/blob/main/config/config.hjson) for more config options). Run `chown lemmy:lemmy /etc/lemmy/ -R` to set the correct owner.
+This is the minimal Lemmy config, put this in `/opt/lemmy/lemmy-server/lemmy.hjson` (see [here](https://github.com/LemmyNet/lemmy/blob/main/config/config.hjson) for more config options).
```

Comment:
Link should go to https://github.com/LemmyNet/lemmy/blob/main/config/defaults.hjson